NEWS.md

```diff
@@ -34,6 +34,8 @@ This release mostly focusses on an overhaul of how testthat works with condition
   should be lazily registered, e.g. with `vctrs::s3_register()`. This
   is useful for introducing an experimental error class without
   encouraging users to depend on the class in their tests.
+  
+* Testthat now respects options(warn = -1) to ignore all warnings.
```

Comment:
Can you please acknowledge yourself and the issue as in https://style.tidyverse.org/news.html#acknowledgement